packages/safe-tools-client/app/templates/application.hbs

```diff
@@ -18,9 +18,7 @@
         boxel-org-title-text-transform='none'
       }}
     >
-      <Boxel::Button @kind='primary'>
-        Connect Wallet
-      </Boxel::Button>
+      <ConnectButton @onConnect={{set this 'isShowingConnectModal' true}} />
```

Comment:
Does the control panel's wallet connection button also trigger opening of the `NetworkConnectModal`?

Reply:
I’ve replaced that button now, thank you